### PR TITLE
Chore/setup express server

### PR DESCRIPTION
**Pull request by awuorm**
#### Any background context you want to provide?

- [ ] I have performed a self-review of my own code
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works .
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by awuorm** [comment]
> 
> 
> @awuorm Nice job on setting up the server so fast! I think you're missing the nodemon dev dependencies

Nice catch! added it.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/LABS-EU3/hackton-backend/issues"
+  },
+  "homepage": "https://github.com/LABS-EU3/hackton-backend#readme",
+  "dependencies": {
+    "cors": "^2.8.5",
```
Comment (mikeattara) [inline package.json]: Missing `pg` package

**Comment by awuorm** [comment]
> 
> 
> I think you are missing the Postgres npm package
We will add it when seting up the db

File: package.json (hunk 1 above)
Comment (mikeattara) [reply package.json]: It should be part of the initial setup IMO

Comment (awuorm) [reply package.json]: I'll add it then

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es6": true
```
Comment (PascalUlor) [inline .eslintrc.js]: add this to the env
"node": true,
    "jest": true

File: .eslintrc.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaFeatures": {
+            "jsx": true
+        },
+        "ecmaVersion": 2018
+    },
+    "plugins": [
```
Comment (PascalUlor) [inline .eslintrc.js]: remove plugins

File: .eslintrc.js (hunk 2 above)
Comment (awuorm) [reply .eslintrc.js]: added

**Pull request merged**